fdb-extensions/src/main/java/com/apple/foundationdb/synchronizedsession/package-info.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * package-info.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2019 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Basic classes to support running operations in synchronized sessions
+ * TODO @see SynchronizedSession
```

Comment:
Maybe the instead of the `TODO` here, it should say that the `SynchronizedSession` class (which can't be a link yet) will be moved here in the next minor version. It's repeated other places, but it might be good to have here as well.